src/routes.js

```diff
@@ -660,6 +659,41 @@ const routes = [
     }
   }),
 
+  asyncRoute({
+    path: '/projects/:projectId([1-9]\\d*)/forms/:xmlFormId/submissions/:instanceId/edit',
+    component: 'FormSubmission',
+    name: 'WebFormEditSubmission',
+    props: true,
+    loading: 'page',
+    meta: {
+      standalone: true,
+      title: () => [form.nameOrId],
+    }
+  }),
+  asyncRoute({
+    path: '/projects/:projectId([1-9]\\d*)/forms/:xmlFormId/submissions/new',
+    component: 'FormSubmission',
+    name: 'WebFormNewSubmission',
+    props: true,
+    loading: 'page',
+    meta: {
+      standalone: true,
+      title: () => [form.nameOrId],
+    }
+  }),
+  asyncRoute({
+    path: '/f/:path(.*)',
+    component: 'FormSubmission',
+    name: 'WebFormPublicLink',
+    props: true,
+    loading: 'page',
+    meta: {
+      standalone: true,
+      requireLogin: false,
+      title: () => [form.nameOrId]
+    }
```

Comment:
If this route is exclusively for public links, I think we should specify `restoreSession: false` in order to reduce the number of requests and to speed up Frontend's initial load time. In that case, there should always be an `st` query parameter. This is also something we could come back to in the future.